docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -318,7 +316,19 @@ spec:
 
   telemetry:
     enabled: false
-    template: {}
```

Comment:
According to tests performed just today, this should work by just removing this line, instead of adding all the components lines. I consider it more readable in this case.

See https://github.com/openstack-k8s-operators/data-plane-adoption/pull/525